api/v1beta2/foundationdbcluster_types.go

```diff
@@ -779,6 +779,8 @@ const (
 	SidecarUnreachable ProcessGroupConditionType = "SidecarUnreachable"
 	// PodPending represents a process group where the pod is in a pending state.
 	PodPending ProcessGroupConditionType = "PodPending"
+	// PodInitializing represents a process group where the pod is in an initializing state.
+	PodInitializing ProcessGroupConditionType = "PodInitializing"
```

Comment:
What is the benefit of this new condition?

Reply:
After some more testing looks like this is not necessary. So I will remove it